force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,11 +67,18 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if(num1 == null || num2 == null){
+            largestNum = null;
+        }else if(num1>num2){
```

Comment:
These will be easier to read with some spaces between the control characters `} else if (num1 > num2) {`